draft-demarco-oauth-status-attestations.md

```diff
@@ -129,7 +148,16 @@ presentation, or situations where the Verifier should not be allowed to
 check the status of a Digital Credential over time due to some privacy constraints,
 in compliance with national privacy regulations.
 
-For instance, consider a scenario under the European Union's General Data Protection Regulation (GDPR), where a Verifier's repeated access to a Status List to check the revocation status of a Digital Credential could be deemed as excessive monitoring of the End-User's activities. This could potentially infringe upon the End-User's right to privacy, as outlined in Article 8 of the European Convention on Human Rights, by creating a detailed profile of the End-User's interactions and credential usage without explicit consent for such continuous surveillance.
+For instance, consider a scenario where a Verifier's repeated access to a
+Status List to check the revocation status of a Digital Credential could
+be deemed as excessive monitoring of the End-User's activities.
+This could potentially infringe upon the End-User's right to privacy,
+as outlined in [Article 8 of the European Convention on Human Rights]
+(https://www.echr.coe.int/documents/convention_eng.pdf) and
```

Comment:
we should move these to informative references

Reply:
It may be resolved by commit https://github.com/peppelinux/draft-demarco-oauth-status-attestations/pull/42/commits/82be3b6d5af95e2146bcb24c5aca642a56e4dfb7

Reply:
@SaraConsoliACN Ories is saying that instead using a markdown href it would be better to use the informative reference section if I get it right

@marinaado WDYT?

Reply:
moved here: https://github.com/peppelinux/draft-demarco-oauth-status-attestations/issues/44